theoplayer/api-reference/roku.md

```diff
@@ -65,7 +71,7 @@ The following key/value pairs are supported on the `contentProtection` attribute
 
 To play videos protected using VUDRM, you need to supply a token. Replace the token `vualto-demo|2018-06-19T09:18:24Z|YSnJPmEceo` in the code below with your own token as well as the associated values of the keys `src` and `licenseUrl`.
 
-```brightscript
+```
```

Comment:
Why was this removed? BrightScript syntax highlighting works just fine [on the live site](https://www.theoplayer.com/docs/theoplayer/api-reference/roku/#content-protection).
```suggestion
```brightscript
```